src/components/CategoryDropdown.jsx

```diff
@@ -26,19 +26,19 @@ const CategoryDropdown = () => {
     <div className="dropdown">
       <button className="dropbtn">Categories</button>
       <div className="dropdown-content">
-        <ul className="dropdown-content">
+        <div className="dropdown-content">
```

Comment:
I think this still functions like a list and would ave left this as is, also its changes that aren't for this ticket